### PR TITLE
status changing function

### PR DESCRIPTION
**Pull request by boazco**
dao and service
Its seems like it will be in self use. so I didnt see why should I add API call for it

### CODE REVIEW

File: src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt [hunk 1]
```diff
@@ -71,4 +71,12 @@ class FeedbackDao(private val sql: DSLContext) {
...
             ?: throw BadRequestException("feedback not found")
     }
+
+    fun changeStatus(feedbackId: UUID, status: Status, activeUser: ActiveUser): Int {
+        return sql.update(feedbackTables)
+        .set(feedbackTables.status, status.toString())
+            .where(feedbackTables.feedbackId.eq(feedbackId))
+            .and(feedbackTables.companyId.eq(activeUser.companyId))
+            .execute()
+    }
```
Comment (elijulian) [inline src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt]: i'm not sure that it's the feedback of the active user that will be the one that needs the status to be updated

Comment (boazco) [reply src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt]: Im not using tha active user id
thats the feedback id and company id

File: src/main/kotlin/com/hibob/academy/filters/AuthenticationFilter.kt [hunk 2]
```diff
@@ -36,7 +36,7 @@ class AuthenticationFilter : ContainerRequestFilter {
                     Role.valueOf(it["role"]!!.uppercase()),
                     Department.valueOf(it["department"]!!.uppercase())
                 )
-            requestContext.setProperty("activeUser", activeUser)
+            requestContext.setProperty(activeUserPropertyName, activeUser)
```
Comment (elijulian) [inline src/main/kotlin/com/hibob/academy/filters/AuthenticationFilter.kt]: related?

File: src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt [hunk 3]
```diff
@@ -27,7 +24,7 @@ class FeedbackDao(private val sql: DSLContext) {
         }
     }
 
-
+    private val employeeTable = EmployeesDao.EmployeesTable.instance
```
Comment (elijulian) [inline src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt]: related?

**Review by elijulian — APPROVED** [review]

**Pull request merged**